sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -206,7 +206,7 @@ def created_on(self):
         return self._created_on
 
     @property
-    def last_udpated_on(self):
+    def last_updated_on(self):
```

Comment:
nit: You might want to include this as a bug fix on the changelog since it looks like part of the public surface.

Reply:
Thanks for your review @kristapratico ! Have updated the changelog. :)